### ISSUE TITLE
ArXiv.org feed support

### ISSUE THREAD

**Issue by v-iashin** [ekarbe/rss-reader#4]
Hi. I tried the extension today and found out that it doesn't work with the arXiv.org RSS feeds.

Settings:
```
{
    "RSSReader.Feeds": [
        {
            "identifier": "GN",
            "title": "Google News",
            "url": "http://arxiv.org/rss/cs.CV"
        }
    ]
}
```

About VSCode section:
```
Version: 1.41.1
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:57:51.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.2.0
```

**Comment by ekarbe** [comment]
Thanks for the report. 

The feed is in RDF schema which is currently not covered by the reader. I will take a look into it and probably fix it with the next version.

**Comment by ekarbe** [comment]
@v-iashin I added support for this feed. It will be released with version 1.2.0 after some testing at the end of the week.
